Godeps/Godeps.json

```diff
@@ -40,53 +35,53 @@
 		},
 		{
 			"ImportPath": "github.com/coreos/fleet/client",
-			"Comment": "v0.3.2-976-g96d7f98",
-			"Rev": "96d7f986de72c3fca04b627ddd6578f7206cb7e4"
+			"Comment": "v0.9.0",
```

Comment:
It's autogenerated; that's how `godep` likes to track individual packages. (And in this case there is no actual code in `github.com/coreos/fleet`, so it's not a Go package strictly speaking, neither is `github.com/deis/deis`.)